src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -82,6 +82,41 @@ public void Await()
         }
 
         [Fact]
+        public void DisplayResults()
```

Comment:
Naming (perhaps something along the lines of DisplayResultsWithCurrentUICulture?)